### PR TITLE
Add new browser tweaks and remove deprecated info

### PR DESCRIPTION
**Pull request by zenithium**
## Description

Resolves: #1594 and Part 1&2a of #1430 

Removed the sentence about redundancy because it is not truly redundant (i.e. privileged pages), and explaining to uBO users how to adjust their settings to handle privileged pages would be too complicated; the filter list overlap is not that important.

Moved the thing about DoH and stuff to the bottom for aesthetic reasons.

I wasn't sure how to handle the bundle of prefetch tweaks, whether to: 
1. Put them all one after the other with an explanation at the bottom
2. Write a description for each (even though the nuances are barely important IMO) and take up a bunch of space
3. Make a sublist about prefetch and just put them all together

After a bunch of thinking I went with 3. since it seemed like the most elegant and logical option but if anyone has a better way feel free to change it ^ ^

- Netlify preview for the mainly edited page: https://deploy-preview-1772--privacytools-io.netlify.com/browsers/#about_config

### CODE REVIEW

**Comment by dngray** [comment]
I'm thinking we should add [Temporary Containers](https://github.com/stoically/temporary-containers) and link to [this article](https://medium.com/@stoically/enhance-your-privacy-in-firefox-with-temporary-containers-33925cd6cd21) under the power user section.

For the uMatrix description we should describe that you don't need NoScript in addition to uMatrix, if you configure uMatrix to [How to block 1st party scripts everywhere by default](https://github.com/gorhill/uMatrix/wiki/How-to-block-1st-party-scripts-everywhere-by-default).

We should also mention for Decentraleyes that [might need further configuration](https://git.synz.io/Synzvato/decentraleyes/-/wikis/Frequently-Asked-Questions#for-umatrix-and-ublock-origin-non-easy-mode-users) if used in conjunction with uMatrix or uBlock.

**Comment by zenithium** [comment]
> I'm thinking we should add [Temporary Containers](https://github.com/stoically/temporary-containers) and link to [this article](https://medium.com/@stoically/enhance-your-privacy-in-firefox-with-temporary-containers-33925cd6cd21) under the power user section. Mention that Cookie Autodelete is not necessary when using.
> 
> I also think [CleanURLs](https://addons.mozilla.org/firefox/addon/clearurls/) is certainly worth adding, typically as people make the mistake of pasting URLs they've visited with those tracking parameters on them.
> 
> For the uMatrix description we should describe that you don't need NoScript in addition to uMatrix, if you configure uMatrix to [How to block 1st party scripts everywhere by default](https://github.com/gorhill/uMatrix/wiki/How-to-block-1st-party-scripts-everywhere-by-default).
> 
> We should also mention for Decentraleyes that [might need further configuration](https://git.synz.io/Synzvato/decentraleyes/-/wikis/Frequently-Asked-Questions#for-umatrix-and-ublock-origin-non-easy-mode-users) if used in conjunction with uMatrix or uBlock.
> 
> How I have mine configured (obviously a very much power user setup):
> 
>     * [dngray/ghacks-user.js/tree/fx-desktop#addons](https://github.com/dngray/ghacks-user.js/tree/fx-desktop#addons)
> 
>     * [dngray/ghacks-user.js/tree/fx-android#addons](https://github.com/dngray/ghacks-user.js/tree/fx-android#addons)
> 
>     * Useful information about addons: [4.1 Extensions (ghacks-user.js)](https://github.com/ghacksuserjs/ghacks-user.js/wiki/4.1-Extensions#small_orange_diamond-extensions-in-no-particular-order)

IMO this should be done in a different PR addressing #1328 specifically as there are many issues with the add-ons section. This PR is for about:config specifically. I could add it in but I am generally against big PRs.

**Comment by dngray** [comment]
IMO this should be done in a different PR addressing #1328 specifically as there are many issues with the add-ons section.

Ah yes, I agree.

File: _includes/sections/browser-tweaks.html [hunk 1]
```diff
@@ -101,14 +95,32 @@ <h3>Getting started:</h3>
...
+    <li>network.dns.disablePrefetchFromHTTPS = true</li>
+    <li>network.predictor.enabled = false</li>
+    <li>network.predictor.enable-prefetch = false</li>
+  </dd>
 
   <dt>network.IDN_show_punycode = true</dt>
   <dd>Not rendering IDNs as their Punycode equivalent leaves you open to phishing attacks that can be very difficult to notice. <a href="https://krebsonsecurity.com/2018/03/look-alike-domains-and-visual-confusion/#more-42636">Source</a></dd>
 
+  <dt>Looking for TRR, DoH or ESNI?</dt>
+  <dd>They have moved to <a href="/providers/dns/#icanndns">our DNS page</a>.</dd>
```
Comment (Mikaela) [inline _includes/sections/browser-tweaks.html]:
```suggestion
  <dd>They have moved to <a href="/providers/dns/#dns">our DNS page</a>.</dd>
```

The link has changed

**Comment by Thorin-Oakenpants** [comment]
change the order so 1) its alphabetical, 2) true vs false logic is split
```
 <ul>
    <li>network.dns.disablePrefetch = true</li>
    <li>network.dns.disablePrefetchFromHTTPS = true</li>
    <li>network.predictor.enabled = false</li>
    <li>network.predictor.enable-prefetch = false</li>
    <li>network.prefetch-next = false</li>
  </dd>
```

**Comment by zenithium** [comment]
I guess if the only problem was the ordering then everything else is fine :smile: I just put them in haphazardly because you said so in the issue
> just plonk it anywhere, we'll deal with the order/sectioning later

**Comment by Thorin-Oakenpants** [comment]
I guess the SB binary description is OK (it's ambiguous IMO, but not sure if I could do better: it's still ambiguous without going into more detail), but that's what part 4 is for

>**Part 4** : Sectionize
> ...
> * at the same time we fixup descriptions and explain the breakage or why there are dragons

The rest looks fine - it'll all get revisited anyway

**Comment by Thorin-Oakenpants** [comment]
> just plonk it anywhere, we'll deal with the order/sectioning later

That's the ordering of the "items" - a clump of prefs like what you did is one item. I was talking about the ordering within your clump :)

**Comment by zenithium** [comment]
Btw @Thorin-Oakenpants while I was writing this commit I remembered that privacy.trackingprotection.enabled enables DNT and this option is also enabled in ghacks-user.js. I was curious what your thought process was on this since AFAIK most sites don't respect DNT and it can even be used to fingerprint

**Comment by Thorin-Oakenpants** [comment]
the mozilla guys (the ones I spoke with in person, who run ETP etc) would like to get rid of it, the tor guys (the ones I spoke with in person) hate it - everyone agrees it's a failed experiment - but it's tied to a number of things in FF, such as ETP, PB mode, and privileged domains - so it's not trivial to remove, and it's low priority - basically no one cares (and that's how I feel about it as well)

AFAIC everyone is already highly fingerprintable: you control 90% of FPing by controlling third parties and scripts. Yes, I know DNT is also an http header - but if they don't detect you're in PB mode, you look like all those who use PB mode (I read a study somewhere that 1/3rd of FF users use a PB window: i.e use, not use exclusively). So I think the numbers are fine.

Furthermore (not sure what the hold up is: there's a bugzilla somewhere), but ETP was/is going to flipped on in normal windows: currently on Nightly 76 it still says TP only for private windows. Anyway, when that happens, everyone will end up the same re DNT. Meanwhile: people also turn on TP in all windows (like you recommend) - but IDK if that makes a significant difference.

Mozilla also respects DNT on AMO etc for those who don't want to ~~fuck~~ mess with privileged domains etc. And lastly, it might even help, what with all the privacy issues in Europe and the GDR...forgot the acronym

tl;dr: it can't hurt tracking wise, it probably doesn't hurt FPing wise

**Comment by zenithium** [comment]
Cool, good to know. Yeah the issue in OP was specifically about tracking on AMO since WebExtensions are disabled there and I think that TP is just the easiest way to fix that since DNT is respected on AMO. The other way would be setting `extensions.webextensions.restrictedDomains` and `privacy.resistFingerprinting.block_mozAddonManager` but I think the former is more elegant and doesn't expose AMO to all addons and throw warnings anytime extensions are installed...

**Comment by Thorin-Oakenpants** [comment]
> Yeah the issue in OP...

I stayed away. I'm done with explaining and justifying everything to everyone. But the proposal is just plain wrong. Something like that is a security loss vs a perceived privacy gain. It shouldn't be mentioned anywhere near the level of PTIO readers

**Comment by Mikaela** [comment]
Thank you @zenithium for the PR and @Thorin-Oakenpants for the review :purple_heart:
